frontend/src/components/Timer/Timer.js

```diff
@@ -0,0 +1,64 @@
+import React from "react";
+
+class Timer extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
```

Comment:
Should we keep seconds as ints everywhere since we plan to count down? Technically js will convert it, but it's good practice to stick to ints if you're doing mathifications. : p